src/wallet/components/WalletDropdownFundLinkEOAWallet.tsx

```diff
@@ -0,0 +1,38 @@
+import { useMemo } from 'react';
+import { useAccount } from 'wagmi';
+import { getOnrampBuyUrl } from '../../fund/utils/getOnrampBuyUrl';
+import { useOnchainKit } from '../../useOnchainKit';
+import type { WalletDropdownFundLinkReact } from '../types';
+import { WalletDropdownFundLinkButton } from './WalletDropdownFundLinkButton';
+
+export function WalletDropdownFundLinkEOAWallet({
+  ...props
+}: WalletDropdownFundLinkReact) {
+  const { projectId, chain: defaultChain } = useOnchainKit();
+  const { address, chain: accountChain } = useAccount();
+
+  // If the connected wallet's active chain is not included in the Wagmi config, accountChain will be undefined. If this
+  // is the case, fall back to the default chain specified in the OnchainKit config.
+  const chain = accountChain || defaultChain;
+
+  if (projectId === null || address === undefined) {
+    return null;
+  }
+
+  const fundingUrl = useMemo(() => {
+    return getOnrampBuyUrl({
+      projectId,
+      addresses: { [address]: [chain.name.toLowerCase()] },
+    });
+  }, [projectId, address, chain]);
+
+  return (
+    // The Coinbase Onramp widget is not very responsive, so we need to override the popup size.
+    <WalletDropdownFundLinkButton
+      {...props}
+      popupHeightOverride={720}
+      popupWidthOverride={460}
```

Comment:
will this only work at these sizes?  would be nice to respect popupSize in both these implementations.  Is this viewable on mobile?

Reply:
Unfortunately yeah, the popup size doesn't even matter, Onramp's view is a hardcoded size, so it'll either have a bunch of empty space, or a bunch of overflow if the popup size doesn't match.

I don't think anyone should be using popup on mobile...

Reply:
we'll have to communicate somehow that popupSize is ignored for EOA Wallets as otherwise this will seem like a bug

Reply:
@alessey yeah good call, I added a jsdoc note to the type, will also add it to the doc site later.